### ISSUE TITLE
Episode: cool gems

### ISSUE THREAD

**Issue by samumantha** [coderefinery/train-the-trainer#90]
This is new. 
Everyone is asked during registration, if they would like to share something cool, so could everyone of us.

**Comment by samumantha** [comment]
@everyone : What would you like to share beyond the CodeRefinery trainer materials?

**Comment by samumantha** [comment]
We did not get much input for this via the registration, so I will ask this again via email closer to the event and explain a bit more. 

Any ideas on how to make it " less scary " and more inviting very welcome :)

**Comment by samumantha** [comment]
From co-working session: Share something from a course you liked

**Comment by samumantha** [comment]
Currently proposed topics:
PO: sure, but I cannot think of any. Can use CFF but its not made by me 
EC: Yes, I can talk about how I used audio data online for my course https://smc-aau-cph.github.io/SPIS 
AVC: I would be able to share about learner personas I use to design a set of teaching material (which I already have prepared). Or things I recently learned during last three months delivering them (which I expect to have as part of my post-training task by that day)) 	
MM: Yes with 50% probability (it might be that what I want to talk about is just already known and widely discussed and I am not aware of that, I need to do some research first) 
RB: I could talk about how I share history and/or how I plan to containerize the teaching environment. 
AVM, RD: Yes
